templates/pages.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:template-part {"slug":"header-site-title","theme":"alphagov-wordpress-theme"} /-->
+
+<!-- wp:post-content {"layout":{"type":"constrained"}} /-->
```

Comment:
Does this allow the background color of blocks inside the post content to span the entire page width? Or does the entire page content have to fit in the 1200px width?